src/pharmpy/modeling/summary_individuals.py

```diff
@@ -0,0 +1,108 @@
+from typing import Union, Callable
+
+import pandas as pd
+import numpy as np
+
+from pharmpy.tools.modelsearch.tool import ModelSearchResults
+from pharmpy.model import Model
+
+from pharmpy.modeling.ml import predict_outliers
+from pharmpy.modeling.ml import predict_influential_individuals
+
+def summary_individuals(results: ModelSearchResults) -> Union[pd.DataFrame,None]:
+    """Creates a summary dataframe keyed by model-individual pairs for an input
+    ModelSearchResults object.
+
+    Parameters
+    ----------
+    results : ModelSearchResults
```

Comment:
If only a startmodel and all candidate models are used these should be the parameters instead of the ModelSearchResults. That way the function would be more general.